pkg/cron_workflow.go

```diff
@@ -570,14 +566,15 @@ func (c *Client) cronWorkflowSelectBuilder(namespace string, workflowTemplateUid
 }
 
 func (c *Client) cronWorkflowSelectBuilderNamespaceName(namespace string, uid string) sq.SelectBuilder {
-	sb := sb.Select("cw.id", "cw.created_at", "cw.uid", "cw.name", "cw.workflow_template_version_id").
+	sb := sb.Select("cw.id", "cw.created_at", "cw.uid", "cw.name", "cw.workflow_template_version_id", "cw.namespace").
```

Comment:
should use method here to get the columns as mentioned in [this comment](https://github.com/onepanelio/core/pull/226/files#r428964039)

Reply:
Makes sense. The mentioned method is equal to what this one does, so I removed it and used that one.